openshift_cli_installer/libs/user_input.py

```diff
@@ -39,6 +40,7 @@ def __init__(self, **kwargs):
         self.aws_access_key_id = self.user_kwargs.get("aws_access_key_id")
         self.aws_secret_access_key = self.user_kwargs.get("aws_secret_access_key")
         self.aws_account_id = self.user_kwargs.get("aws_account_id")
+        self.gcp_service_account_file = self.user_kwargs.get("gcp_service_account_file")
```

Comment:
moved this because in next line `self.clusters` is assigned to `get_clusters_from_user_input()`, and to add `gcp_service_account_file` in clusters dict (as did for aws values), this needs to be assigned before that.
This is for GcpIpiCluster class.